cli/command/telemetry.go

```diff
@@ -72,6 +73,15 @@ func (cli *DockerCli) MeterProvider(ctx context.Context, opts ...sdkmetric.Optio
 	return sdkmetric.NewMeterProvider(allOpts...)
 }
 
+// GetDefaultMeter gets the default metric.Meter for the application
+// using the given metric.MeterProvider
+func GetDefaultMeter(mp metric.MeterProvider) metric.Meter {
+	return mp.Meter(
+		"github.com/docker/cli",
+		metric.WithInstrumentationVersion(version.Version),
+	)
+}
+
```

Comment:
Is it possible to move this to a less prominent location? It's my opinion that this function should only be invoked within this module and I'd like to avoid us accidentally calling it from a plugin.